code/modules/power/apc.dm

```diff
@@ -1252,6 +1254,18 @@
 			charging = 0
 			chargecount = 0
 
+		if(excess >= 5000000 && !shock_proof) //If there's more than 5,000,000 watts in the grid, start arcing and shocking people.
+			if(prob(5))
+				var/list/shock_mobs = list()
+				for(var/C in view(get_turf(src), 5)) //We only want to shock a single random mob in range, not every one.
+					if(iscarbon(C))
+						shock_mobs +=C
+				if(shock_mobs.len)

```

Comment:
its' a fine proc, but I didn't want it bouncing around, zapping machines, causing EMPs, and the likes.